main.go

```diff
@@ -391,6 +518,9 @@ func _main() error {
 	}
 
 	resultFile := ec.GetPanelResultsFile(project.Id, panel.Id)
+	if cache {
+		defer os.Remove(resultFile)
```

Comment:
You can just remove this in all cases even if it's redundant. But you should leave a comment as to why it's necessary